packages/grid/test/grid-wrapper.test.js

```diff
@@ -0,0 +1,56 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
+import '../vaadin-grid.js';
+import '../vaadin-grid-column-group.js';
+import '../vaadin-grid-column.js';
+import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { flushGrid } from './helpers.js';
+
+class GridWrapper extends PolymerElement {
+  static get template() {
+    return html`
+      <vaadin-grid id="grid">
+        <vaadin-grid-column-group>
+          <slot name="group"></slot>
+        </vaadin-grid-column-group>
+        <slot></slot>
+      </vaadin-grid>
+    `;
+  }
+}
```

Comment:
Rather extend `HTMLElement`, so this can be re-used with the Lit tests.

```suggestion
class GridWrapper extends HTMLElement {
  constructor() {
    super();
    this.attachShadow({ mode: 'open' }).innerHTML = `
      <vaadin-grid id="grid">
        <vaadin-grid-column-group>
          <slot name="group"></slot>
        </vaadin-grid-column-group>
        <slot></slot>
      </vaadin-grid>
    `;
  }
}
```

Reply:
Done.